essay.tex

```diff
@@ -0,0 +1,64 @@
ing on a project and thus can cover several disciplines, however it seems to me that it would be quite redundant for typical AAA Development as staff are normally very specialised and won't normally cross over to other disciplines in normal development practice. To further back this up, I again turn to Source 2 even goes as to say that "Prioritization of user-stories is a difficult task in agile environment because of its volatile nature." \cite{popli2014prioritising}, which is especially true of games development as there are many factors that can affect the length of a sprint, as mentioned by Al-azawi, Ayesh and Obaidy, such as "Feature Creep" \cite{al2014towards} which entails "new functionality is added during the development phase to increase project scope and change schedule time" \cite{al2014towards} and these type of events are what I would describe as volatile and such can affect the sprint length on a project, along with causing other issues such as "increased error, possible defects and increased chances of failure" \cite{al2014towards} which again in turn can also lead to increased sprint time, but then again, they only state that this is applicable to "Unmanaged" \cite{al2014towards} Feature Creep, and there are many case studies into methods that are meant to keep feature creep to a minimum during development, such as this one by Scott Crabtree, who outlines many methods of reducing feature creeps effects such as "Work through the design as thoroughly as you can, as early as you can"\cite{featurecreep}, which echos back to another method of game development which is Waterfall, where a very large design document is created at the start of a project, outlining all of the features the game will have, however as with all collaborative projects, especially on the scale of AAA game development, different parties will make changes and not update the document, so features go unwritten but are planned to be implemented which creates unnessecary work and strain on the team, which is why Agile Development techniques or atleast work ethics from the Agile Philosophy are now being implemented into games development, along with software development, which has been embracing Agile for longer than the games industry and when research was carried out by Pettrillo and Pimenta, they found that "good practices adopted in the traditional software industry are also found in the games industry"\cite{petrillo2010agility}, which points to a positive as surely the methods used in traditional development carry should carry over to games, however they go on to say "In both studies, the quality of the team was dominant for the success of the project"\cite{petrillo2010agility}, again pointing back to my earlier point that outside factors such as team structure and specialisation are more important to a project,  which is why we'll now take a look closer at the way methods used in regular development and see whether they have any significance in Games Development.
```

Comment:
This section could be further decomposed into multiple paragraphs to aid the flow of reading.